test/abstractarray.jl

```diff
@@ -1286,20 +1286,24 @@ end
     ax=axes(a,1)
     @test a==a[ax]
     a=reshape(collect(1:100),(10,10))
-    @test a==a[ax,ax]
+    @test a[:,1]==a[ax,1]
```

Comment:
This test is not for the method that you've modified here, right?

Reply:
Yes, it should use linear indexing even for multidimensional arrays, e.g. `@test a[2:5] == 2:5`

Reply:
Yes, it should use linear indexing even with multidimensional arrays, e.g., `@test a[2:5] == 2:5`.

Reply:
Changes did as asked.